students/funcs1.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+
+def my_sum(first, second):

```

Comment:
File placed incorrectly. It should be located inside your working dir: `students/krokos/1`